llama-index-integrations/embeddings/llama-index-embeddings-openai/llama_index/embeddings/openai/base.py

```diff
@@ -367,6 +355,16 @@ def _get_aclient(self) -> AsyncOpenAI:
             self._aclient = AsyncOpenAI(**self._get_credential_kwargs(is_async=True))
         return self._aclient
 
+    def _create_retry_decorator(self):
+        """Create a retry decorator using the instance's max_retries."""
+        return create_retry_decorator(
+            max_retries=self.max_retries,
+            random_exponential=True,
+            stop_after_delay_seconds=60,
+            min_seconds=1,
+            max_seconds=20,
```

Comment:
This is kind of nice. I wonder if its worth making these other params configurable? Or nah?

Reply:
I think so, but probably later.

Also, I found out this problem is repeated on other embedding as well, and probably more wherever the `create_retry_decorator ` is used.

I'm currently working on separate PRs for Upstage and Anyscale embedding,
https://github.com/search?q=repo%3Arun-llama%2Fllama_index%20max_retries%3D6&type=code

Reply:
BTW I probably need help to add test on this one.